### DISCUSSION TITLE
404 at working pages in localhost

### DISCUSSION THREAD

**Discussion by danilo-leal** [vercel/next.js#20187] [Help]
Hey guys!

So I have my working website deployed with Vercel at the link: https://dl-co.vercel.app/

What's kicking with me is that I have two specific routes that are working fine in localhost but if you would go to check them out in the link above, it pops a 404. You can check my repository to dive a bit deep in how I'm calling these pages out, it's in /pages/index.js. They are the **_iFood.js_** and **_Rapiddo.js_** ones. 

Does anyone have any idea what's going on? I'm a bit clueless since the other pages are working just fine and the code hasn't any typo. Kinda confused here. 

The weird thing is that the pages files locally here are named like I've written above: **iFood** and **Rapiddo**. But if you see [at the repository](https://github.com/danilo-leal/daniloleal.co/tree/main/pages/cases), both are lower case. I'm guessing maybe that's simply it but I have no clue why this mess of names is happening.

**Comment by jamesmosier** [comment]
The issue is that Git does not recognize file case changes by default. Windows & Mac OS are case insensitive so that is why you see them in 1 case but on GitHub you see them as another. You may have committed these as one case and then changed them to another case, but Git did not pick up this change. This [Stack Overflow](https://stackoverflow.com/questions/17683458/how-do-i-commit-case-sensitive-only-filename-changes-in-git) post talks a bit about it as well.

Therefore to fix this you should do the following:

1. Delete `rapiddo.js` and `ifood.js`
2. Commit the deleted files
3. Create 2 new files named in whatever case you want them to be in
4. Commit those 2 files with the new names
